coil-base/src/main/java/coil/fetch/UriFetcher.kt

```diff
@@ -42,6 +44,18 @@ internal class UriFetcher(
         size: Size,
         options: Options
     ): FetchResult {
+        if (data.scheme == ContentResolver.SCHEME_ANDROID_RESOURCE) {
+            val resourceId = extractResourceId(data)
+            resourceId?.let {
+                val drawable = context.getDrawableCompat(resourceId)
```

Comment:
If I understand correctly, this code part assumes that `resourceId` belongs to same applications. However `Uri` can point to `resourceId` which belongs to different application:
```
"android.resource://{pkgName}/{resourceId}"
```

Reply:
Thanks, @audkar for pointing this issue. I got the solution from [Glide source code](https://github.com/bumptech/glide/blob/d8278f43a94484f20822170b71ac7b66269a2aa6/library/src/main/java/com/bumptech/glide/load/resource/drawable/ResourceDrawableDecoder.java) 
I will do the changes ASAP :blush: